compiler/driver.ml

```diff
@@ -378,10 +381,14 @@ let driver source_file (options : Cli.options) : int =
     0
   with
   | Errors.StructuredError (msg, pos) ->
+    let bt = Printexc.get_raw_backtrace () in
```

Comment:
It's part of 49efb5ddd7f8ee8b108c4455cad97078e266748e.

The `Format` module uses exceptions internally, so if you do some formatting _then_ try to get the backtrace, it will often point to nonsensical parts of `Format`.

Reply:
Wouahou much OCaml engineering such good error messages